### DISCUSSION TITLE
Linux Support

### DISCUSSION THREAD

**Discussion by FruityWelsh** [SitinCloud/Owlyshield#36] [Ideas]
What are some blockers or issues expected in trying to port to Linux? Where can community members help?

This project seems very cool to me, but for my personal use I only need Linux antivirus support, and for work I would need a service that supports both. I am sure others would be interested as well!

**Comment by dlescos** [comment]
Hi,

We are not far from releasing the Linux version. It uses BPF (through the very nice [refbpf crate](https://github.com/foniod/redbpf/) to hook VFS events.

Stay tuned!

**Reply by FruityWelsh** [reply]
That's awesome, that seems like the best place to hook into from my limited knowledge too. Really looking forward to this release!

**Reply by dlescos** [reply]
Many thanks!

Have a look at the new Linux branch (unmature).
